app/models/api_key.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class ApiKey < ApplicationRecord
+  include PaperTrailTraceable
+
+  belongs_to :organization
+
```

Comment:
I would also add a validation of value presence here? what do you think?

Reply:
I’ve tried to do add it but it doesn’t look great.

We cannot add `validates :value, presence: true`  with a `before_create` callback that sets the value. The value for new records will be always missing.
If we change callback to `before_validation` we can add a validation but it’s not possible to test, there will no ability to set `nil` value and receive an error that value is missing because of callback. With such setup the validation is a redundancy.

The most reasonable way of adding it by only applying it to update action. But honestly, I'd skip it at all for this particular case.
Let me know if you think I missed something.